specs/customizability.md

```diff
@@ -0,0 +1,350 @@
+- `timestamp` is the number of seconds since UNIX epoch
+- `network_version` is used by the FVM to select gas pricing policy
+- `base_fee` is measured in *atto* and represents the base price for gas
+- `power_scale` is the number of decimals to take into account from the FIL token collateral balance when converting it into voting power expressed as `u64`, which is what CometBFT expects. For example if the scale is 0 then every 1 FIL gives 1 voting power, if it’s 3 then every 0.001 FIL does, and if it’s -1 then every 10 FIL. The power is rounded upwards, so that we don’t end up with 0 power, which would be rejected by CometBFT; for example if the scale is 1, then both 1.1 FIL and 1.9 FIL give 2 power.
+- `validators` is a list of stakes of each genesis validator, given by the token collateral and their public key.
+- `accounts` is a list of genesis balances, given by tokens and FVM addresses.
+- `eam_permission_mode` defines who can deploy smart contracts, which could be anyone, or a set of whitelisted addresses.
+- `ipc` is only enabled if we want Fendermint to participate in an IPC hierarchy:
+    - `gateway` defines the parameters of the IPC `Gateway` actor:
+        - `subnet_id` defines the full `SubnetID` from the root
+        - `bottom_up_check_period` is the expected checkpoint frequency for the subnet. It is important that subnet nodes create checkpoints in the ledger at the same heights; doing otherwise would be a consensus failure.
+        - `majority_percentage` is the checkpoint quorum size expressed as a number between 0 and 100; it should be at least 67 to be Byzantine fault tolerant.
+        - `active_validators_limit` is the number of validators who can participate in the subnet consensus; it is important that both the parent and the child subnet agree on this value, so they select the same top validators the same way.
+
+The `genesis.json` file is usually constructed in one of two ways:
+
+- Using the `fendermint genesis` [CLI commands](https://github.com/consensus-shipyard/ipc/blob/specs/fendermint/app/options/src/genesis.rs) to incrementally add validators and accounts as described [here](https://github.com/consensus-shipyard/ipc/blob/specs/docs/fendermint/running.md#genesis).
+- Using the `fenderming genesis ipc from-parent` command to fetch the parameters from the subnet actor on the parent subnet and export them to file.
+
+# Gas policy
+
+The gas price in the FVM is by default [determined](https://github.com/filecoin-project/ref-fvm/blob/c39d880d086aa2e771c7190163436e02715d80f3/fvm/src/machine/mod.rs#L156) by the network version when the `NetworkConfig` is created, but could further be customised by assigning to the `price_list` field. Should Fendermint have to do that, it would be in the [constructor](https://github.com/consensus-shipyard/ipc/blob/7af25c4c860f5ab828e8177927a0f8b6b7a7cc74/fendermint/vm/interpreter/src/fvm/state/exec.rs#L132) of the `FvmExecState`.
```

Comment:
> assigning to the `price_list` field

`price_list` is hypothetical here or where does it come from?

Reply:
The FVM will return a `price_list` for the specified Filecoin network version, but the user can customize it before creating an FVM Machine.